packages/react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -702,40 +604,38 @@ describe('ComboBox', () => {
     >();
     const initialOption = { key: '1', text: 'Text' };
 
-    safeMount(<ComboBox options={[initialOption]} autoComplete="on" allowFreeform onChange={onChange} />, wrapper => {
-      const inputElement = wrapper.find('input');
-      inputElement.simulate('input', { target: { value: 't' } });
-      inputElement.simulate('input', { target: { value: 'te' } });
-      inputElement.simulate('input', { target: { value: 'tex' } });
-      inputElement.simulate('input', { target: { value: 'text' } });
-      inputElement.simulate('keydown', { which: KeyCodes.enter });
-      expect(onChange).toHaveBeenCalledTimes(1);
-      expect(onChange.mock.calls[0][1]).toEqual(initialOption);
-      expect(onChange.mock.calls[0][2]).toEqual(0);
-      expect(onChange.mock.calls[0][3]).toEqual(initialOption.text);
+    const { getByRole } = render(
+      <ComboBox options={[initialOption]} autoComplete="on" allowFreeform onChange={onChange} />,
+    );
 
-      wrapper.update();
-      expect(wrapper.find('input').props().value).toEqual('Text');
-    });
+    const combobox = getByRole('combobox');
+    userEvent.type(combobox, 'text{enter}');
+    expect(onChange).toHaveBeenCalledTimes(1);
+    expect(onChange.mock.calls[0][1]).toEqual(initialOption);
+    expect(onChange.mock.calls[0][2]).toEqual(0);
+    expect(onChange.mock.calls[0][3]).toEqual(initialOption.text);
+    expect(combobox.getAttribute('value')).toEqual('Text');
   });
 
   it('merges callout classNames', () => {
-    safeMount(
+    mockCreatePortal();
```

Comment:
I'm pretty sure instead of mocking createPortal you can do something like this. `baseElement` is `document.body` by default. https://testing-library.com/docs/react-testing-library/api/#baseelement
```tsx
const { baseElement, getByRole } = render(...);
...
const callout = baseElement.querySelector('.ms-Callout');
```